statsmodels/discrete/count_model.py

```diff
@@ -713,7 +713,7 @@ class ZeroInflatedPoissonResults(CountResults):
     @cache_readonly
     def _dispersion_factor(self):
         mu = self.predict(which='linear')
-        w = 1 - self.predict() / np.exp(self.predict(which='linear'))
+        w = 1 - self.fittedvalues / np.exp(self.predict(which='linear'))
```

Comment:
+1 for this change.  Confident you made this optimization in all the appropriate locations?

Reply:
I'm pretty sure that there are more outside of discrete in the generic code.
I only did a search for 'predict()' in discrete_model and count_model.

Reply:
If I track them down and point them out, mind adding them in this PR?  Follow-up PRs are liable to languish.